<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Led" name="toolButton_sensor11">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>420</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="Led" name="toolButton_sensor12">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>320</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="Led" name="toolButton_valve1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="cedaTank" name="toolButton_tank1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>310</y>
      <width>171</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>tank 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>valve1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>sensor11</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>sensor12</string>
    </property>
   </widget>
   <widget class="Led" name="toolButton_tank1_overflow">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>350</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>sensorOverflow</string>
    </property>
   </widget>
   <widget class="Led" name="toolButton_sm1_led_error">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>390</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>390</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ledV1Error</string>
    </property>
   </widget>
   <widget class="cedaTank" name="toolButton_tank2">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>171</width>
      <height>140</height>
     </rect>
    </property>
    <property name="text">
     <string>tank 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>sensor22</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>130</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>sensor21</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>60</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>sensorOverflow</string>
    </property>
   </widget>
   <widget class="Led" name="toolButton_tank2_overflow">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>60</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="Led" name="toolButton_sm2_led_error">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>100</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>100</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ledP1Error</string>
    </property>
   </widget>
   <widget class="Led" name="toolButton_sensor21">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>130</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="Led" name="toolButton_sensor22">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="Led" name="toolButton_pump1">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>440</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>440</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>pump1</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>440</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>20</width>
      <height>411</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>20</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>160</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_tank2_consume">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>220</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cold Water Consumption</string>
    </property>
   </widget>
   <widget class="cedaTank" name="toolButton_tank3">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>90</y>
      <width>81</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>tank 3</string>
    </property>
   </widget>
   <widget class="Led" name="toolButton_valve2">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>140</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>120</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>valve2</string>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>150</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>150</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>140</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>sensor31</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>sensor32</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ledV2Error</string>
    </property>
   </widget>
   <widget class="Led" name="toolButton_sm3_led_error">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>120</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="Led" name="toolButton_sensor31">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>140</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>80</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>sensorOverflow</string>
    </property>
   </widget>
   <widget class="Led" name="toolButton_tank3_overflow">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>80</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="Led" name="toolButton_sensor32">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>100</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="Led" name="toolButton_resistor1">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>160</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>160</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>resistor1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>160</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>sensorTemp</string>
    </property>
   </widget>
   <widget class="Led" name="toolButton_sensorTemp">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>160</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>160</y>
      <width>3</width>
      <height>61</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Orientation::Vertical</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_tank3_consume">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>220</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Hot Water Consumption</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>1020</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ledR1Error</string>
    </property>
   </widget>
   <widget class="Led" name="toolButton_sm4_led_error">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>180</y>
      <width>22</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdNumber_temp">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>330</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>true</bool>
    </property>
    <property name="digitCount">
     <number>4</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_19_lcd_temp_display">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>290</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature Display</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar_temp">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>370</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scenarioA">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Scenario A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20_scenarioA">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>45</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Complete Filling</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21_scenarioB">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>85</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Partial Equilibrium</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scenarioB">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Scenario B</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22_scenarioC">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>125</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Backflow Protection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scenarioC">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Scenario C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scenarioD">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Scenario D</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23_scenarioD">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>165</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Decision with Limited Information</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_scenarioE">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Scenario E</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24_scenarioE">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>205</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dry Heating Protection</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Led</class>
   <extends>QToolButton</extends>
   <header>led.h</header>
  </customwidget>
  <customwidget>
   <class>cedaTank</class>
   <extends>QToolButton</extends>
   <header>cedatank.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
